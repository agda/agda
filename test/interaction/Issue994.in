top_command (cmd_load currentFile [])
IOTCM currentFile Interactive Indirect (Cmd_refine 0 (intervalsToRange Nothing [Interval () (Pn () 20 2 7)  (Pn () 27 2 14)]) "λ x → x")
