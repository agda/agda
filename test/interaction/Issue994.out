(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set â†’ Set" nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-highlight-load-and-delete-action)
(agda2-give-action 0 'no-paren)
(agda2-status-action "")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
